subsys/bluetooth/mesh/shell.c

```diff
@@ -1632,7 +1632,7 @@ static int mod_pub_get(const struct shell *shell, uint16_t addr, uint16_t mod_id
 static int mod_pub_set(const struct shell *shell, uint16_t addr, uint16_t mod_id,
 		       uint16_t cid, char *argv[])
 {
-	struct bt_mesh_cfg_mod_pub pub;
+	struct bt_mesh_cfg_mod_pub pub = {0};
```

Comment:
I like to initial struct's to {0}, but is it really necessary here. The uuid is set to NULL in line 1641.

Reply:
True, I missed that because NCS has an old version. Sorry for that. I'll close the PR.